### PR TITLE
Nrn segment section distance scaler

### PR DESCRIPTION
**Pull request by alejoe91**
Added a `NrnSegmentSectionDistanceScaler` in the `parameterscalers.py` to generalize the behavior of the `NrnSegmentSomeDistanceScaler`. 

In brief, it works as the soma scaler, but one can indicate a `ref_section` and a `ref_location`. In our case, we need to define the AIS distributions with respect to section `ais[0](0)`. Then we would instantiate this with `ref_section="ais[0]"` (this is a string used to identify the right section), and `ref_location=0.0`.

### CODE REVIEW

File: bluepyopt/ephys/parameterscalers.py [hunk 1]
```diff
@@ -196,3 +198,124 @@ def __str__(self):
...
+        self.ref_sec = ref_sec
+
+        if not (0.0 <= self.ref_location <= 1.0):
+            raise ValueError("ref_location must be between 0 and 1.")
+
+        if self.dist_param_names is not None:
+            for dist_param_name in self.dist_param_names:
+                if dist_param_name not in self.distribution:
+                    raise ValueError(
+                        'NrnSegmentSomaDistanceScaler: "{%s}" '
```
Comment (darshanmandge) [inline bluepyopt/ephys/parameterscalers.py:258]: `'NrnSegmentSectionDistanceScaler: "{%s}" ' `
instead of 
`'NrnSegmentSomaDistanceScaler: "{%s}" '`

Comment (alejoe91) [reply bluepyopt/ephys/parameterscalers.py:258]: @darshanmandge you know you can make code suggestions that I can automatically commit?? :)

Comment (alejoe91) [inline bluepyopt/ephys/parameterscalers.py:258]:
```suggestion
                        'NrnSegmentSectionDistanceScaler: "{%s}" '
```
@darshanmandge  With the +- button, like this!

Comment (darshanmandge) [reply bluepyopt/ephys/parameterscalers.py:258]: Thanks. This feature is useful to know.

**Pull request merged**